internal/pkg/scaffold/helm/chart.go

```diff
@@ -134,7 +147,20 @@ func CreateChart(projectDir string, opts CreateChartOptions) (*scaffold.Resource
 	if err != nil {
 		return nil, nil, err
 	}
-	log.Infof("Created %s/%s/", HelmChartsDir, c.GetMetadata().GetName())
+
+	relChartPath := filepath.Join(HelmChartsDir, c.GetMetadata().GetName())
+	absChartPath := filepath.Join(projectDir, relChartPath)
+	if err := fetchChartDependencies(absChartPath); err != nil {
```

Comment:
Fetching chart dependencies may seem unrelated to this PR, but I was testing it against [charts in the stable repo](https://github.com/helm/charts/tree/master/stable) and noticed that it failed to generate the manifest for many of them because their dependencies were missing.